src/shared/email/templates/welcome.html

```
@@ -0,0 +1,132 @@
+            <p class="message">Ready to start learning?</p>
+            
+            <a href="{{frontendUrl}}/dashboard" class="button">Go to Dashboard</a>
+        </div>
+        
+        <div class="footer">
+            &copy; 2025 GenAI LMS. All rights reserved.
+        </div>
+    </div>
+</body>
+</html>
+</html>
```

HTMLHint (1.5.0)

[error] Tag must be paired, no start tag: [ </html> ]

(tag-pair)